src/main/java/com/google/sps/data/Recipe.java

```diff
@@ -8,14 +8,8 @@
   public static final String TIMESTAMP_KEY = "timestamp";
   public static final String VOTES_KEY = "votes";
 
-  public String title;
+  public RecipeMetadata metadata;
```

Comment:
Recipe should not extend DBObject as it now contains an id through RecipeMetadata.